src/Razor/src/Microsoft.VisualStudio.Editor.Razor/DefaultVisualStudioRazorParser.cs

```diff
@@ -383,7 +383,7 @@ private void QueueChange(SourceChange change, ITextSnapshot snapshot)
         {
             _dispatcher.AssertForegroundThread();
 
-            _latestChangeReference = _parser.QueueChange(change, snapshot);
+            _latestChangeReference = _parser?.QueueChange(change, snapshot);
```

Comment:
Add a comment on why this could be null. Looking at the code it looks like it can never be null.